src/lib/api/pricer/getLlamaPrice.ts

```diff
@@ -31,5 +31,5 @@ export async function getLlamaPrice({
     `https://coins.llama.fi/prices/current/${llamaKey}`
   )
   const parsed = respSchema.parse(resp.data)
-  return parsed.coins[llamaKey].price
+  return parsed?.coins[llamaKey].price
```

Comment:
This will always be defined because `parse` will throw if it fails to validate against the schema.

Reply:
yes